src/components/DpDateRangePicker/DpDateRangePicker.stories.mdx

```diff
@@ -0,0 +1,24 @@
+import { Meta } from '@storybook/addon-docs'
+import DpDateRangePicker from './'
+
+<Meta
+    title="Components/DateRangePicker"
+    component={DpDateRangePicker}
+/>
+
+# DateRangePicker
+
+The DateRangePicker component allows users to enter two dates within a given range either by typing
+or by selecting a date from the calendar.
+
+Use the date range picker when you want to provide a view that allows the user to select two different calendar dates.
```

Comment:
While i initially brought up the need to express the "When to use", i feel that this is redundant with the first sentence...

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/524/commits/4414ba27a3d77bf0ef3abb7e6c77cfd712ea8951